ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,245 @@
+
+The workflow process can be tracked to check the status of a whole process or of individual actions, and to solve any problems that may occurred as a result of a misconfiguration or any other external issues.
+
+
+## Check the process status
+
+The *OVERVIEW* tab in the *Processes* menu entry displays a list of all processes that are currently being executed or have been executed in the system. Per default, all existing processes are displayed but can also be filtered according to different criteria.
+
+[comment]: <> (UI info -> columns, filter, etc.)
+
+A *VIEW* can also be created to customize the display options. For detailed information, see [Link](#to-be-completed.).
+
+
+### Prerequisites
+
+- A workflow is created, see [Create a workflow](01_ManageWorkflows.md#create-a-workflow).
+- A workflow process has been initiated, see [Link](#to-be-determined).
+
+### Procedure
+
+*Workflows > Menu entry Processes > Tab OVERVIEW*
+
+![Processes](/Assets/Screenshots/ActindoWorkFlow/Workflows/Processes.png "[Processes]")
+
+1. Click *Status* drop-down list to filter the processes according to their status, if necessary.   
+All existing processes with the selected status are displayed.
+
+  The following statuses are available:
+
+  - **All**  
+  All existing processes are displayed. This option is selected per default.
+
+  - **Ready**  
+  The processes that are ready are displayed.
+
+  - **In progress**  
+  The currently active processes are displayed.
+
+  - **Error**  
+  The processes that have produced an error are displayed.
+
+  - **Done**  
+  The processes that are finished are displayed.
+
+  - **Dead**  
+  The processes that are no longer running, probably due to a misconfiguration, are displayed.
+
+  - **Suspended**
+
+  - **Aborted**
+
+[comment]: <> (Process elements/UI folder? Link to UI - Oli: Meaning of Ready, Suspended and Aborted)
+
+2. Alternatively, click the *Workflow* drop-down list to display a specific workflow type.
+All existing processes of the selected workflow type are displayed.
+
+  > [Info] Processes can also be filtered with the filter options. Click the ![Filter](/Assets/Icons/Filter.png "[Filter]") (Filter) button to display the filter options. For detailed information, see [Link](#to-be-completed).
+
+3. Check the current process status in the *Status* column.
+
+4. Click the checkbox to select the desired process to be checked.  
+The toolbar is displayed.
+
+[comment]: <> (Stimmt das? Was macht der Checkbox links?)
+
+5. Alternatively, click a process to select it and to check individual process actions.  
+A read-only diagram of the process including all individual places and actions is displayed. For detailed information, see [Check the process action status](#check-the-process-action-status).     
+
```

Comment:
möglicher ablauf dieser procedure:
- filter nach status auf all (ich kenne den status meines prozesses ja nicht und möchte alle prozesse anzeigen)
- filter nach workflow (denn ich möchte ja den status eines bestimmten prozesses checken)
- prozess in liste suchen (ggf. filtermöglichkeiten der spalten nutzen) und status checken
- prozess in liste anklicken, um zu sehen, wo der prozess genau steht (hinweis/verweis auf die anderen procedures zum check der process actions und tokens)